Sources/RxAlamofire.swift

```diff
@@ -901,6 +927,53 @@ extension Reactive where Base: DataRequest {
   }
 }
 
+// MARK: Reactive + DownloadRequest
+
+extension Reactive where Base: DownloadRequest {
+    
+  public func response() -> Single<DefaultDownloadResponse> {
+    return Single.create { cb in
+      let request = self.base.response {
+        cb(.success($0))
+      }
+      return Disposables.create(with: request.cancel)
+    }
+  }
+    
+  public func responseSerialized<Serializer: DownloadResponseSerializerProtocol>(
+    queue: DispatchQueue? = nil,
+    responseSerializer: Serializer
+  )
+    -> Single<DownloadResponse<Serializer.SerializedObject>>
+  {
+    return Single.create { cb in
```

Comment:
I like that we use `Single`, but I think the rest of the API is using `Observable`, right? in that case we might want to comply and use `Observable` as well, or think about a way to distinguish between `Single`/`Observable` api